documentation/configuration.markdown

```diff
@@ -568,10 +52,7 @@ Puppet Server relies on `logrotate` to manage the log file, and installs a confi
 
 ### HTTP Traffic
 
-Puppet Server logs HTTP traffic in a format similar to Apache, and to a separate
-file than the main log file. By default, this is located at
-`/var/log/puppetlabs/puppetserver/puppetserver-access.log` (open source releases) and
-`/var/log/pe-puppetserver/puppetserver-access.log` (Puppet Enterprise).
+Puppet Server logs HTTP traffic in a format similar to Apache, and to a separate file than the main log file. By default, this is located at `/var/log/puppetlabs/puppetserver/puppetserver-access.log` (open source releases) and `/var/log/pe-puppetserver/puppetserver-access.log` (Puppet Enterprise).
```

Comment:
This path is now the same for both OSS Puppet Server 2.x and PE 2015.2.x - `/var/log/puppetlabs/puppetserver/puppetserver-access.log`.